opennms-doc/releasenotes/src/asciidoc/releasenotes/whatsnew-20.adoc

```diff
@@ -13,21 +13,73 @@
 * link:https://issues.opennms.org/browse/NMS-9303[NMS-9303]: Deprecated Capsd and Linkd event definitions are removed.
   Otherwise unused event definitions are also cleaned up and removed from the code and configurations.
   For a detailed list of all removed events see link:https://issues.opennms.org/browse/NMS-9303[NMS-9303].
-  For people maintaining databases prior OpenNMS 1.8, we provide a maintenance script for Linux and Windows in the _opennms-contrib_ package to clean up old events in the database.
-  The script is named `NMS-9303-cleanup-deprecated-events.sh}` and executes a SQL script.
+  For people maintaining databases prior to OpenNMS 1.8, we provide a maintenance script for Linux and Windows in the _opennms-contrib_ package to clean up old events in the database.
+  The script is named `NMS-9303-cleanup-deprecated-events.sh` and executes a SQL script.
   In case you run PostgreSQL on Windows, you can use `psql.exe -d opennms -U opennms -W -a -f NMS-9303-cleanup-deprecated-events.sql` to run the maintenance script.
 
+=== Future Breaking Changes
+
+* *Configuration File Format*: In OpenNMS Horizon 21, we will (finally) be re-formatting all XML
```

Comment:
Speaking of format, shouldn't we stick to the one sentence per line in these files too? :)